src/index.ts

```diff
@@ -18,14 +18,19 @@ export {
   CloudEventV1Attributes,
   Version,
   ValidationError,
+  // From message
+  Headers,
+  Mode,
+  Binding,
+  Message,
+  Deserializer,
+  Serializer,
+  HTTP,
   // From transport
   Emitter,
   Receiver,
-  Mode,
   Protocol,
   TransportOptions,
-  Headers,
-  headersFor,
```

Comment:
If we are going to release this as 3.2.0, then i think we need to still export `headersFor` and not remove it until 4.0

Reply:
Ahh - good point...